corehq/apps/export/views.py

```diff
@@ -570,6 +570,15 @@ def page_context(self):
 
         return context
 
+    # Add the output of djng_current_rmi to view context, which requires having
+    # the rest of the context, specifically context['view'], available.
+    # See https://github.com/jrief/django-angular/blob/master/djng/templatetags/djng_tags.py
+    def get_context_data(self, **kwargs):
+        context = super(BaseDownloadExportView, self).get_context_data(**kwargs)
+        from djangular.templatetags.djangular_tags import djng_current_rmi
+        context['djng_current_rmi'] = json.loads(djng_current_rmi(context))
+        return context
```

Comment:
Is this something that should always be available?  Will we need it any time we want to have fully external JS and RMI?  If so, I'm wondering if we should make our own subclass of `JSONResponseMixin` (with a better name), and include this there.